src/Breadcrumb/Breadcrumb.scss

```diff
@@ -62,20 +61,25 @@
     }
   }
 
-  ol.is-mobile {
-    flex-direction: row-reverse;
-    justify-content: flex-end;
+  ol {
+    display: flex;
```

Comment:
a small refactoring to avoid using CSS utility classes, git makes too big of a diff out of it 🙂